gpflux/layers/basis_functions/fourier_features/random/base.py

```diff
@@ -74,7 +79,12 @@ def _sample_students_t(nu: float, shape: ShapeType, dtype: DType) -> TensorType:
 
 class RandomFourierFeaturesBase(FourierFeaturesBase):
     def __init__(self, kernel: gpflow.kernels.Kernel, n_components: int, **kwargs: Mapping):
-        assert isinstance(kernel, RFF_SUPPORTED_KERNELS), "Unsupported Kernel"
+        assert isinstance(
+            kernel, (RFF_SUPPORTED_KERNELS, RFF_SUPPORTED_MULTIOUTPUTS)
+        ), "Unsupported Kernel"
+        if isinstance(kernel, RFF_SUPPORTED_MULTIOUTPUTS):
+            for k in kernel.latent_kernels:
+                assert isinstance(k, RFF_SUPPORTED_KERNELS), "Unsupported Kernel"
```

Comment:
It would be better to give a different warning message than the one above so a user can see which check is violated.